src/app.ts

```diff
@@ -19,7 +20,11 @@ export function createApp(
     const config = createProxyConfig(options);
     const client = unleashClient || new Client(config);
 
-    const proxy = new UnleashProxy(client, config);
+    const openApiService = new OpenApiService(config);
+
+    const proxy = new UnleashProxy(client, config, openApiService);
+
+    openApiService.useDocs(app);
```

Comment:
Should there be a flag to toggle the OpenAPI docs endpoints, like on the backend?

Reply:
Great point! I'll do something about that. Got any preference as to whether the default is on or off?

Reply:
Looks like it defaults to off on the backend, could probably reuse the env var name as well.

https://github.com/Unleash/unleash/blob/main/src/lib/create-config.ts#L314-L315